packages/shared/trade/src/transaction-sidebar/transaction-sidebar.tsx

```diff
@@ -127,17 +174,16 @@ export const TransactionSidebar = ({
       handleSubmit={handleSubmit}
       canSubmit={canSubmit && !leverageDisabled}
       handleLeverUpToggle={handleLeverUpToggle}
-      onCancelCallback={onCancelCallback}
+      onCancelCallback={handleActionSidebarCancel}
       leveredUp={leveredUp || false}
       showLeverUpToggle={!pathname.includes('lend')}
       leverageDisabled={leverageDisabled}
       showDrawer={isPortfolio ? false : showDrawer === true}
-      hideTextOnMobile={isPortfolio ? false : true}
+      hideTextOnMobile={isPortfolio || !hideTextOnMobile ? false : true}
     >
       {children}
-      {riskComponent || <LiquidationRisk state={state} />}
```

Comment:
If you were running into type issues, you can just do this here instead of having to change all the other files:
`<LiquidationRisk state={state as TradeState} />`

Reply:
Do you mean BaseTradeState?